packages/zosjobs/__tests__/__system__/ModifyJobs.system.test.ts

```diff
@@ -31,9 +30,9 @@ describe("Modify Jobs - System Tests", () => {
     beforeAll(async () => {
         testEnvironment = await TestEnvironment.setUp({
             testName: "zos_modify_jobs"
-        });
+        }, REAL_SESSION = await TestEnvironment.createSession());
+
```

Comment:
Similar to previous comment...

in the afterAll, we have a cancelJobs.cancelJob...

Should we just add the iefbr14Job to the list of resources and let the cleanUp take care of it?

I don't feel strongly about this one. If it isn't broken... 😋